src/components/Dashboard.tsx

```diff
@@ -85,17 +95,22 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
   )
 }
 
-const FilterComponent: React.FC<{ template: TemplateDetails; filter: Filter }> = ({
-  template,
-  filter,
-}) => {
+const FilterComponent: React.FC<{
+  template: TemplateDetails
+  filter: Filter
+  setFiltersCount: (setState: Function) => void
+}> = ({ template, filter, setFiltersCount: setLoadingFilters }) => {
   const templateType = template.code
   const { loading, applicationCount } = useListApplications({
     type: templateType,
     perPage: 1,
     ...filter.query,
   })
 
+  useEffect(() => {
+    if (loading === false) setLoadingFilters((currentCount: number) => currentCount + 1)
```

Comment:
```suggestion
    if (!loading) setLoadingFilters((currentCount: number) => currentCount + 1)
```
Slightly more typical